google/cloud/forseti/scanner/scanners/blacklist_scanner.py

```diff
@@ -99,11 +100,15 @@ def get_instance_networks_interfaces(self):
 
             for instance_from_data_model in data_access.scanner_iter(
                     session, 'instance'):
-                instance = Instance.from_json(
-                    instance_from_data_model.full_name,
-                    instance_from_data_model.data,
-                    instance_from_data_model.parent.name)
-                network_interfaces.append(instance.create_network_interfaces())
+
+                proj = project.Project(
+                    project_id=instance_from_data_model.parent.name,
+                    full_name=instance_from_data_model.parent.full_name,
```

Comment:
We ran into a problem when executing queries in a yield_per session and that will cause the query to throw exception.
In this case if count(instance_from_data_model) > the PER_YIELD threshold (1024), the session will get only 1024 items and in the first query (but the session is still being used) and when you do 'instance_from_data_model.parent.name / full_name', a new query is generated and run using the same session which will cause the command out of sync error.

A similar issue:
https://github.com/GoogleCloudPlatform/forseti-security/issues/2027
Do you mind spliting that into 2 queries? This will be the same for other scanners.